core/src/main/scala/kafka/log/CleanerConfig.scala

```diff
@@ -37,5 +37,6 @@ case class CleanerConfig(numThreads: Int = 1,
                          maxIoBytesPerSecond: Double = Double.MaxValue,
                          backOffMs: Long = 15 * 1000,
                          enableCleaner: Boolean = true,
-                         hashAlgorithm: String = "MD5") {
+                         hashAlgorithm: String = "MD5",
+                         brokerCompressionType: String = "uncompressed") {
```

Comment:
Thanks for the suggestion. I think CompressionCodec is more apt here though. Will change in next version.